src/Compiler.php

```diff
@@ -579,12 +580,16 @@ private function transformCamelCaseToCSS(string $property): string
 
     private function stripEventHandlers(DOMElement $node)
     {
+        $removeAttributes = [];
         /** @var DOMAttr $attribute */
         foreach ($node->attributes as $attribute) {
             if (strpos($attribute->name, 'v-on:') === 0 || strpos($attribute->name, '@') === 0) {
-                $node->removeAttribute($attribute->name);
```

Comment:
Is not much faster do here removeAttribute rather than put another loop ?

Reply:
The old solution is not working properly in our project. Only the first attribute was removed. Unfortunately I couldn't find out why, so I had this solution, which first collects all attributes and then removes them later.

Reply:
The old solution is not working properly in our project. Only the first attribute was removed. Unfortunately I couldn't find out why, so I had this solution, which first collects all attributes and then removes them later.